src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp

```diff
@@ -30,12 +30,20 @@
 #include "common_test_utils/subgraph_builders/concat_with_params.hpp"
 #include "common_test_utils/subgraph_builders/split_concat.hpp"
 
-#define MARK_MANDATORY_FOR_HW_DEVICE(GET_TEST_NAME)                                                   \
-    [&](const testing::TestParamInfo<PropertiesParams>& info) {                                       \
-        std::string name = GET_TEST_NAME(info);                                                       \
+#define MARK_MANDATORY_PROPERTY_FOR_HW_DEVICE(GET_TEST_NAME)                                          \
+    [](const testing::TestParamInfo<PropertiesParams>& info) {                                        \
+        std::string name = "";                                                                        \
+        if (nullptr != GET_TEST_NAME) {                                                               \
+            name = GET_TEST_NAME(info);                                                               \
+        }                                                                                             \
```

Comment:
This changed is covered by https://github.com/openvinotoolkit/openvino/pull/26250